tst/EnergyPlus/api/TestRuntime.py

```diff
@@ -80,3 +80,12 @@ def error_handler(severity: int, message: bytes) -> None:
 api.runtime.callback_progress(state, progress_handler)
 api.functional.callback_error(state, error_handler)
 api.runtime.run_energyplus(state, sys.argv[1:])
+
+print("MUTING CONSOLE OUTPUT")
+state2 = api.state_manager.new_state()
+api.runtime.clear_callbacks()
+api.functional.clear_callbacks()
+api.runtime.set_console_output_mute_state(state2, True)
+print("RUNNING MUTED ENERGYPLUS...")
+api.runtime.run_energyplus(state2, sys.argv[1:])
+print("MUTED E+ RUN DONE")
```

Comment:
And this Python version works nicely too.  When I am running multiple threads of E+ in a single script this really helps.